.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-Pits
```

Comment:
OK, let's remove this entirely.  If it needs to be applied every time, you would just make the profile active by default.

Reply:
The initial plan was to activate this profile only when needed. I reintroduced it here, not to make it active by default, but rather to override the existing Maven Failsafe Plugin configured in the `codeQuality` profile of the `java-parent` pom.xml, as the integration tests were running without utilizing the `its` profile’s configurations. Making it active by default did not resolve the issue. Therefore, I’m removing the Maven Failsafe Plugin from the `its` profile and relying solely on the one defined in the `codeQuality` profile within the `java-parent` pom.xml. I believe this is the correct approach to avoid conflicts and unpredictable behavior.

Reply:
Instead, using a negated property for default activation worked. Moreover, this approach provides a simple, predictable way to skip the execution of the profile whenever necessary.